comp/core/workloadmeta/collectors/internal/containerd/image_sbom_trivy.go

```diff
@@ -139,3 +96,54 @@ func (c *collector) extractSBOMWithTrivy(_ context.Context, storedImage *workloa
 
 	return nil
 }
+
+func (c *collector) startScanResultHandler(ctx context.Context, resultChan <-chan sbom.ScanResult) {
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case result, ok := <-resultChan:
+		  if !ok { return }
```

Comment:
linter doesn't like it 

```suggestion
		  if !ok { 
		    return 
		  }
```